### ISSUE TITLE
assigning integer to structure should produce syntax error

### ISSUE THREAD

**Issue by arcanist123** [abaplint/abaplint#1981]
this code should create a syntax error
```abap 
 TYPES:
    BEGIN OF ts_str,
      comp_one TYPE i,
    END OF ts_str.
  DATA lv_int1 TYPE i VALUE 1.
  DATA lv_str TYPE ts_str.
  MOVE lv_int1 TO lv_str.
  WRITE lv_str.
```

**Comment by larshp** [comment]
@arcanist123 try again, 2.74.46

**Comment by larshp** [comment]
closing, no reply
